astropy/modeling/functional_models.py

```diff
@@ -2215,10 +2216,10 @@ class Moffat1D(Fittable1DModel):
         plt.show()
     """
 
-    amplitude = Parameter(default=1)
-    x_0 = Parameter(default=0)
-    gamma = Parameter(default=1)
-    alpha = Parameter(default=1)
+    amplitude = Parameter(default=1, description="Amplitude of the model")
+    x_0 = Parameter(default=0, description="X position of maximum of Moffat model")
+    gamma = Parameter(default=1, description="Core width of Moffat model")
+    alpha = Parameter(default=1, description="Power index of the Moffat model")
```

Comment:
This what the description is really good for: `alpha` would not be obvious otherwise. Great!